### PR TITLE
Add 1.7.8.7 informations to vulnerability article

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add 1.7.8.7 informations to vulnerability article
| Fixed ticket? |

### CODE REVIEW

File: news/_posts/2022-07-22-major-security-vulnerability-on-prestashop-websites.md [hunk 1]
```diff
@@ -67,7 +67,9 @@ Consider contacting a specialist to perform a full audit of your site and make s
 
 ## Additional information
 
-A patch version is currently being tested, and should be released soon.
+[PrestaShop 1.7.8.7](https://build.prestashop.com/news/prestashop-1-7-8-7-maintenance-release/) has been released and it strengthens the MySQL Smarty cache storage against code injection attacks.
```
Comment (eternoendless) [inline news/_posts/2022-07-22-major-security-vulnerability-on-prestashop-websites.md]:
```suggestion
[PrestaShop 1.7.8.7](/news/prestashop-1-7-8-7-maintenance-release/) has been released to strengthen the MySQL Smarty cache storage against code injection attacks.
```

File: news/_posts/2022-07-22-major-security-vulnerability-on-prestashop-websites.md [hunk 2]
```diff
@@ -67,7 +67,9 @@ Consider contacting a specialist to perform a full audit of your site and make s
...
 
 We would like to take the opportunity to stress out once more the importance of keeping your system updated to prevent such attacks. This means regularly updating both your PrestaShop software and its modules, as well as your server environment.
 
+
+Note: the original article has been edited on Monday 25 of July 2022 to add up-to-date informations about 1.7.8.7 version.
```
Comment (eternoendless) [inline news/_posts/2022-07-22-major-security-vulnerability-on-prestashop-websites.md]:
```suggestion
Note: the original article has been updated on Monday, July 25, 2022, to add information about the release of PrestaShop 1.7.8.7.
```

**Review by coffrep — COMMENTED** [review]
The new content on the vulnerability article is approved on my end, there are no modifications to make.

**Comment by matks** [comment]
@eternoendless ready to publish

**Comment by eternoendless** [comment]
Thanks @matks

**Pull request merged**